data/_models/shared/Pedigree.csv

```diff
@@ -2,11 +2,12 @@ tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,ref
 Pedigree,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C27957,Pedigree studies in which the individual is part of.,"Patient registry, FAIR Genomes"
 Pedigree,,id,string,1,,,,,,,http://purl.obolibrary.org/obo/NCIT_C204540,An identifier that is used to identify related pedigree members,"Patient registry, FAIR Genomes"
 Pedigree,,alternate ids,string_array,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C90353,Additional identifiers that identify a pedigree,"Patient registry, FAIR Genomes"
+Pedigree,,consanguinity,bool,,,,,,,,http://purl.obolibrary.org/obo/OMIT_0004546,Information on whether the patient is a child from two family members who are second cousins or closer.,"Patient registry, FAIR Genomes"
 Pedigree,,disease,ontology_array,,,CatalogueOntologies,Diseases,,,,http://purl.obolibrary.org/obo/NCIT_C25209,,"Patient registry, FAIR Genomes"
 Pedigree,,members,refback,,,,Pedigree members,,pedigree,,http://purl.obolibrary.org/obo/NCIT_C41256,"List of members of the pedigree. If the current pedigree definition is attached to the proband, it contains the whole list of pedigree members, including the proband. If the definition is attached to an individual different than the proband, it only contains two entries: one that describes that member, e.g. the proband mother or father, and one that points to the proband.","Patient registry, FAIR Genomes"
 Pedigree members,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C41256,"Any of the individuals who are descended from a common progenitor, related by marriage or other legal tie, or by a feeling of closeness.","Patient registry, FAIR Genomes"
 Pedigree members,,pedigree,ref,1,TRUE,,Pedigree,,,,http://purl.obolibrary.org/obo/NCIT_C25364,Identifier of the family,"Patient registry, FAIR Genomes"
 Pedigree members,,individual,ref,1,TRUE,,Individuals,,,,http://purl.obolibrary.org/obo/NCIT_C25364,Identifier of the individual,"Patient registry, FAIR Genomes"
-Pedigree members,,relative,ref,1,TRUE,,Individuals,,,,http://purl.obolibrary.org/obo/NCIT_C25364,Identifier of the individual. If left empty the relation is against the proband,"Patient registry, FAIR Genomes"
+Pedigree members,,relative,ref_array,1,TRUE,,Individuals,,,,http://purl.obolibrary.org/obo/NCIT_C25364,Identifier of the individual. If left empty the relation is against the proband,"Patient registry, FAIR Genomes"
```

Comment:
hmm, so you want to be able to get all sisters in one row? That is okay I suppose

Reply:
Yes, that's the idea. During the RD3 migration process, we imported a new row for each relationship (e.g., patient to mother, mother to patient, patient to sister 1, patient to sister 2, patient to brother 1, patient to brother 2, etc.). We noticed that this becomes redundant (and the table size dramatically increases) when we imported parent to children relationships. We decided to adjust the column type so that we can import one row that a defines relationships between similar family members.